PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -214,12 +217,6 @@ for Hemisphere in L R ; do
       ${CARET7DIR}/wb_command -metric-mask "$AtlasSpaceFolder"/"$NativeFolder"/"$Subject"."$Hemisphere"."$Map"_s"$CorrectionSigma".native.func.gii "$AtlasSpaceFolder"/"$NativeFolder"/"$Subject"."$Hemisphere".roi.native.shape.gii "$AtlasSpaceFolder"/"$NativeFolder"/"$Subject"."$Hemisphere"."$Map"_s"$CorrectionSigma".native.func.gii
```

Comment:
We probably don't need to continue to generate these `_s"$CorrectionSigma"` files, since they are no longer involved in generating the _BC files.  However, maybe the _BC script should do a -cifti-separate at the end to generate the expected gifti versions of the _BC files.

Reply:
Since the gifti output is not necessary every time this module is called, I would choose to add another optional argument `--gifti-output=True` to control this behavior.

Reply:
We may want this to be a deliberate design decision, where we either always generate the gifti versions, or never, modifying the behavior of the existing code.  @glasserm ?

Reply:
Well, I have thought about it, but ended up thinking we should maintain existing behavior for now.

Reply:
So we should NOT add optional argument `--gifti-output=True` and the gifti generation is a separated step outside the module `MyelinMap_BC.sh`?

Reply:
my thought had been that this module would only do the stuff we want done across all usecases and that the surrounding code would be individualized per each use case.